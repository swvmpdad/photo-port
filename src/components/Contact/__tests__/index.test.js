import React from "react";
import { render, cleanup } from "@testing-library/react";
import '@testing-library/jest-dom/extend-expect';
import Contact from '..';

const mockContactSelected = jest.fn();
const mockSetContactSelected = jest.fn();

afterEach(cleanup);

describe('Contact component', () => {
    it('renders', () => {
        render(<Contact 
            contactSelected={mockContactSelected}
            setContactSelected={mockSetContactSelected}
            />);
    });

    it('Matches snapshot', () => {
        const { asFragment } = render(<Contact 
            contactSelected={mockContactSelected}
            setContactSelected={mockSetContactSelected}
            />);
            expect(asFragment).toMatchSnapshot();
    });
});