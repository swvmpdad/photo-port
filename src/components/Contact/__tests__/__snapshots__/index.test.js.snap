// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact component Matches snapshot 1`] = `[Function]`;
